src/test/java/amazin/BaseTest.java

```diff
@@ -94,72 +91,66 @@ protected void testSetField(Fields.Book field, Object val) {
     }
 
     protected void assertFieldBase(Fields.Book field, Object expected, boolean defaultValue,
-                                   TriConsumer<String, Object, Object> assertFunc) {
+            TriConsumer<String, Object, Object> assertFunc) {
         Object result;
         String message = "Unkown field enum specified: " + field + ".";
-        switch(field) {
-            default:
-                result = null;
-
-            case ID:
-                result = book.getId();
-                if(defaultValue)
-                    expected = DEFAULT_ID;
-                break;
-
-            case NAME:
-                result = book.getName();
-                if(defaultValue)
-                    expected = DEFAULT_NAME;
-                break;
-
-            case AUTHOR:
-                result = book.getAuthor();
-                if(defaultValue)
-                    expected = DEFAULT_AUTHOR;
-                break;
-
-            case DESCRIPTION:
-                result = book.getDescription();
-                if(defaultValue)
-                    expected = DEFAULT_DESCRIPTION;
-                break;
-
-            case ISBN:
-                result = book.getISBN();
-                if(defaultValue)
-                    expected = DEFAULT_ISBN;
-                break;
-
-            case PICTURE:
-                result = book.getPicture();
-                if(defaultValue)
-                    expected = DEFAULT_PICTURE;
-                break;
-
-            case PUBLISHER:
-                result = book.getPublisher();
-                if(defaultValue)
-                    expected = DEFAULT_PUBLISHER;
-                break;
-
-            case SKU:
-                result = book.getSKU();
-                if(defaultValue)
-                    expected = DEFAULT_SKU;
-                break;
-
-            case INVENTORY:
-                result = book.getInventory();
-                if(defaultValue)
-                    expected = DEFAULT_INVENTORY;
-                break;
-
-            case PRICE:
-                result = book.getPrice();
-                if(defaultValue)
-                    expected = DEFAULT_PRICE;
-                break;
+        switch (field) {
```

Comment:
Please indent correctly and not add unnecessary commits

Reply:
This is the linting plugin